src/pages/search.js

```diff
@@ -80,7 +167,7 @@ export default withSearch(({ search }) => {
           <h3>Searching...</h3>
         </div>
       )}
-      <div className={searchPageStyle.searchResults}>
+      <div id="searchResults">
```

Comment:
@jasonlcrane this is also a regression against my last commits, could you fix it please?